models/demos/t3000/llama2_70b/tests/perf/test_llama_matmul_perf.py

```diff
@@ -91,12 +91,12 @@ def __init__(self, device):
 
     def __call__(self, x):
         # Assume interleaved input
-        ff_out = tt_lib.operations.primary.matmul_1d(
+        ff_out = ttnn.matmul(
```

Comment:
difference between 1d and general is hidden now?

Reply:
Yes. And tt_lib.operations.primary.matmul_1d will be deleted.